lib/utilshare_web/views/payment_request_view.ex

```diff
@@ -0,0 +1,23 @@
+defmodule UtilshareWeb.PaymentRequestView do
+  use UtilshareWeb, :view
+  alias UtilshareWeb.PaymentRequest
+
+  def render("index.json", %{payment_requests: payment_requests}) do
+    render_many(payment_requests, PaymentRequestView, "expense_instance.json")
+  end
+
+  def render("show.json", %{payment_request: payment_request}) do
+   render_one(payment_request, PaymentRequestView, "payment_request.json")
+  end
+
+  def render("payment_request.json", %{payment_request: payment_request}) do
+    %{
+      id: payment_request.id,
+      percent: payment_request.percent,
+      requester: UtilshareWeb.UserView.render('show.json',%{user: payment_request.requester}),
```

Comment:
I think we should do `user.json` instead of `show.json` -- show will load in the households of the user which seems unnecessary in this content

LGTM other than that

Reply:
will do.